libr/core/cmd_open.c

```diff
@@ -311,7 +311,10 @@ R_API void r_core_file_reopen_debug (RCore *core, const char *args) {
 		r_core_bin_rebase (core, new_baddr);
 		// r_core_bin_load (core, newfile, new_baddr);
 		// reload symbols with new baddr
-		r_core_cmd0 (core, ".isrzM*");
+		r_core_cmd0 (core, ".is*");
```

Comment:
this fixes this specific case, but .isrzM* should work too. i can merge this if we open an issue to fix the real problem